test/statsapi.jl

```diff
@@ -0,0 +1,26 @@
+using Distributions
+using StatsAPI: pvalue
```

Comment:
Would it make sense to have Distributions reexport `pvalue`? I'd be generally in favor but don't feel too strongly.

Reply:
Wouldn't this break HypothesisTests?

Reply:
Not after HypothesisTests extends `pvalue` from StatsAPI.

Reply:
Sure, but all existing releases of HypothesisTests would suddenly be broken, wouldn't they? Certainly, this could be fixed by modifying the registry but I think this makes it less attractive to re-export `pvalue`.

Reply:
I'll merge and release without re-exporting `pvalue` for now but this can be reconsidered and changed in a subsequent release if desired.